pkg/providers/aws/provider_blobstorage.go

```diff
@@ -156,6 +156,49 @@ func (p *BlobStorageProvider) CreateStorage(ctx context.Context, bs *v1alpha1.Bl
 		},
 	}
 
+	// Adding tags to s3
+	p.Logger.Infof("bucket %s found, Adding tags to bucket", *bucketCreateCfg.Bucket)
+
+	// get the environment from the CR
+	defaultOrganizationTag := bs.Spec.Env.Value
+
+	// new session but don't need credentials
+	s3svctag := s3.New(session.Must(session.NewSession(&aws.Config{Region: aws.String(stratCfg.Region)})))
+
+	//get Cluster Id
+	clusterId, _ := resources.GetClusterId(ctx, p.Client)
+	// Tagging input
+	bucketTaggingInput := &s3.PutBucketTaggingInput{
```

Comment:
Reading up about `PutBucketTagging`, it seems to override existing tags if there are any. Do we want this? Would it be safer to get a list of tags and merge our tags with that existing list? 

wdyt @austincunningham @ciaranRoche

Reply:
I do not think we would like to override any existing tags. As we do not know if our SRE team will add their own tags after the fact.

Reply:
My reading of the docs is that tag keys have to be unique so it would not allow over writing. Can't put my hands on where I read that now.

Reply:
found it https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/allocation-tag-restrictions.html